lib/streamlit/__init__.py

```diff
@@ -142,6 +142,7 @@ def _update_logger() -> None:
 bar_chart = _main.bar_chart
 bokeh_chart = _main.bokeh_chart
 button = _main.button
+_internal_button_group = _main._internal_button_group
```

Comment:
Yeah this feels a little bit weird. I have added it here so that tests can use `st._internal_button_group` instead of `ButtonGroupMixin._button_group(st._main, ...)`. Alternatively, we could remove the tests for it altogether and only have the `feedback` tests and respective code coverage. I am happy with either way. Do you have a preference given this context?

Reply:
I have decided to remove the `_internal_button_group` from the `__init__.py` and use `ButtonGroupMixin._button_group(st._main,...)` in the tests. This way, only test files are affected and we do not have noise in the `__init__` 👍